### PR TITLE
Adding a docker-compose file for adding flow data

### PR DESCRIPTION
**Pull request by i3149**
This lets people get started quickly exploring flow in NR.

### CODE REVIEW

File: hostsflow/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    network_mode: host
+
+  ktranslate:
+    image: kentik/ktranslate:v2
+    container_name: ktranslate
+    restart: unless-stopped
+    environment:
+      - NEW_RELIC_API_KEY=${NR_INSIGHTS_INSERT_KEY}
```
Comment (thezackm) [inline hostsflow/docker-compose.yaml:20]: this should be the `${NR_LICENSE_KEY}`

File: hostsflow/README [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Demo for generating flow from a Linux host and displaying it in New Relic.
+
+1) Create a free account at https://newrelic.com/.
+
+2) Follow the guided install for "Network Flow". Copy the account ID and API Key generated.
+
+3) Run with
+
+sudo NETWORK=$MY_NETWORK NR_INSIGHTS_INSERT_KEY=$MY_KEY NR_ACCOUNT_ID=$MY_ACCOUNT_ID docker-compose up
```
Comment (thezackm) [inline hostsflow/README:9]: replace: `NR_INSIGHTS_INSERT_KEY=$MY_KEY`

with `NR_LICENSE_KEY=$MY_KEY`

**Review by thezackm — CHANGES_REQUESTED** [review]

File: hostsflow/README (hunk 2 above)
Comment (i3149) [reply hostsflow/README]: ok -- should be good now.

File: hostsflow/README [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+2) Follow the guided install for "Network Flow". Copy the account ID and API Key generated.
+
+3) Run with
+
+sudo NETWORK=$MY_NETWORK NR_LICENSE_KEY=$MY_KEY NR_ACCOUNT_ID=$MY_ACCOUNT_ID docker-compose up
+
+For example,
+
+sudo NETWORK='^eth' NR_INSIGHTS_INSERT_KEY=NRII-foobarrrr NR_ACCOUNT_ID=3222432 docker-compose up
```
Comment (thezackm) [inline hostsflow/README:13]: sorry! missed this one earlier, should be: 

```
sudo NETWORK='^eth' NR_LICENSE_KEY=a1b2c3d4 NR_ACCOUNT_ID=123456789 docker-compose up
```

**Review by thezackm — CHANGES_REQUESTED** [review]

**Review by thezackm — APPROVED** [review]
🎉 🚀

**Pull request merged**